src/PrestaShopBundle/Form/Admin/Improve/International/Tax/TaxType.php

```diff
@@ -61,8 +61,24 @@ public function __construct(TranslatorInterface $translator)
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $invalidCharsText = sprintf(
+            '%s <>;=#{}',
+            $this->translator->trans('Invalid characters:', [], 'Admin.Notifications.Info')
```

Comment:
Should the placeholder be inside the translation message? @LouiseBonnard

Reply:
Maybe you could add a const `TypedRegexValidator::CATALOG_CHARS` for `<>;=#{}` ? Like you did for name chars

Reply:
Nope, it's fine. ;-)

Reply:
This idea came when was facing errors like - we have one help message but in reality another characters were validated than in the help message of the form. Keeping it in one place, helps avoiding issues of this type.